kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/ExecWebSocketListener.java

```diff
@@ -95,14 +98,13 @@ public void close() {
             if (!executorService.awaitTermination(10, TimeUnit.SECONDS)) {
                 executorService.shutdownNow();
             }
-        } catch (Throwable t) {
-            throw KubernetesClientException.launderThrowable(t);
-        }
-
-        WebSocket ws = webSocketRef.get();
-        try {
-            if (ws != null) {
-                ws.close(1000, "Closing...");
+            if (client != null) {
```

Comment:
But that shuts down the dispatcher for the whole client so no more calls can be made using the client following the websocket close. Client must continue to be usable after websocket close.